app/controllers/recipes_controller.rb

```diff
@@ -0,0 +1,6 @@
+class RecipesController < ApplicationController
+  def show 
+    @recipe = Recipe.find(params[:id])
+    @ingredient = RecipeIngredient.where(recipe: @recipe.id)
```

Comment:
This works, however it would be little more efficient AND would make your controller a little thinner to just do `@recipe.ingredients` in the view. You will get the same results as your code on line 4.

Reply:
I get this now, after looking at it later this week. Thank you!